org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/AcquireRequestHandler.java

```diff
@@ -63,12 +63,12 @@ public AcquireRequest process(AcquireRequest bean) throws EventException {
 
 			bean.setStatus(Status.COMPLETE);
 			bean.setMessage(null);
-		} catch (ScanningException | InterruptedException e) {
+		} catch (Exception e) {
 			e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)